deflatebench.py

```diff
@@ -623,18 +626,6 @@ def main():
             print("Error, parameter '--gen' conflicts with parameters '--single' and '--multi'")
             sys.exit(1)
 
-    if args.minigzip:
-        if args.minideflate:
-            print("Error, parameter '--minigzip' conflicts with parameter '--minideflate'")
-            sys.exit(1)
-        cfgRuns['testtool'] = 'minigzip'
-
-    if args.minideflate:
-        if args.minigzip:
-            print("Error, parameter '--minideflate' conflicts with parameter '--minigzip'")
-            sys.exit(1)
-        cfgRuns['testtool'] = 'minideflate'
```

Comment:
I haven't tested this yet, but I don't see where args.testtool gets inserted into cfgRuns['testtool']

Reply:
Good catch. It must have been due to a bad merge. I had to redo some of the commits.